doc/ref_arch/openstack/chapters/chapter01.rst

```diff
@@ -103,11 +103,39 @@ Wallaby."
 Principles
 ----------
 
-OpenStack Reference Architecture must obey to the following set of
-principles described in:
+Architectural principles
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-- :ref:`cntt:common/chapter00:anuket general principles`
-- :ref:`cntt:common/chapter00:architectural principles`
+This OpenStack Reference Architecture must obey to the following
```

Comment:
"must obey: is rather strong -- can we instead use "should conform to", "should adhere to", ...
```suggestion
This Reference Architecture for OpenStack based Cloud Infrastructure must obey the following
```

Reply:
@pgoyal01 Thanks, approved